Makefile

```diff
@@ -6,7 +6,9 @@ else
 	out-dir := $(CURDIR)/out
 endif
 
+ifneq ($(wildcard $(TA_DEV_KIT_DIR)/host_include/conf.mk),)
 include $(TA_DEV_KIT_DIR)/host_include/conf.mk
```

Comment:
Replacing these three lines with
`-include $(TA_DEV_KIT_DIR)/host_include/conf.mk`
would give the same result